integration_tests/tests/cucumber.rs

```diff
@@ -68,6 +70,8 @@ use crate::utils::{
 
 pub const LOG_TARGET: &str = "cucumber";
 pub const LOG_TARGET_STDOUT: &str = "stdout";
+const BLOCK_REWARD: u64 = 5_000_000;
```

Comment:
this does not seem correct?
How did you calculate this?
This is supposed to be a decaying nonconstant number + varying fee

Reply:
I did get it from the JS version. Take a look at `integration_tests/helpers/constants.js`

Reply:
There is a very good chance this might only work under very strict circumstances. The JS code was old, and not always maintained. Perhaps see if you can calculate it some other way, perhaps using the constants directly. Using RUST we now have access to that code. 

Although this is still a strange way of calculating the number of blocks mined.

Reply:
Refactored code to use ConsensusConstants directly and compute block reward from height